.github/workflows/e2e_tests.yml

```diff
@@ -18,8 +18,8 @@ jobs:
           node-version: '10.16.0'
       - run: npm ci
       - run: npm install jest --global
-      - run: npm run dist
```

Comment:
I moved this down, after `test:e2e-docker-up`, because the `test:e2e-docker-up` might take a while. While that's completing, we can build the package in the meantime.
This should save us some time on the duration of the workflow, as well as reduce errors while calling `test:e2e-docker-ping`